### PR TITLE
feat(storage): Added encrypted storage endpoint

### PR DESCRIPTION
**Pull request by rjchow**
Adding encrypted storage functionality to enable transfers.
endpoints:

POST /storage/create

    takes a valid open-attestation file and returns the file id and key necessary for decrypting it

GET /storage/:id

    returns the ciphertext for the associated file id

Used OpenPGP.js for easy compatibility reasons, has a type parameter in the results so that new cryptographic schemes can be easily added.

### CODE REVIEW

File: src/storage/documentService/documentService.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+const verify = require("@govtechsg/oa-verify");
+const uuid = require("uuid/v4");
+const { encryptString } = require("./crypto");
+const config = require("../config");
+const { put, get, remove } = require("../dynamoDb");
+
+const DEFAULT_TTL = 1000 * 60 * 5; // 5 Minutes
```
Comment (yehjxraymond) [inline src/storage/documentService/documentService.js]: Thought we decided to use like 1 hour or something? (I know i wrote this)

File: src/storage/documentService/documentService.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+const getDocument = async (id, { cleanup }) => {
+  const params = {
+    TableName: process.env.OA_DOC_STORAGE_TABLE,
+    Key: {
+      id
+    }
+  };
+  const document = await get(params);
+  if (cleanup && cleanup === "true") {
```
Comment (yehjxraymond) [inline src/storage/documentService/documentService.js]:
```suggestion
  if (cleanup) {
```

If it's a string, should handle at the event handler level, its just weird here.

File: src/storage/documentService/documentService.js (hunk 1 above)
Comment (rjchow) [reply src/storage/documentService/documentService.js]: changed to 1hr

File: src/storage/documentService/documentService.js (hunk 2 above)
Comment (rjchow) [reply src/storage/documentService/documentService.js]: moved to event handler

**Pull request merged**